src/tr1/game/game_string.def

```diff
@@ -18,3 +18,11 @@ GS_DEFINE(MISC_EMPTY_SLOT_FMT, "- EMPTY SLOT %d -")
 
 GS_DEFINE(CONSOLE_HELP_DEBUG, "Toggles visual debug information.")
 GS_DEFINE(CONSOLE_HELP_VSYNC, "Toggles vertical sync.")
+
+GS_DEFINE(GAMEPLAY_SETTINGS_FIX_BEAR_AI_DESCRIPTION,                    "Fixes the bear pat attack so it does not miss Lara.")
+GS_DEFINE(GAMEPLAY_SETTINGS_FIX_SHOTGUN_TARGETING_DESCRIPTION,          "Makes the shotgun no longer fire when a locked target moves out of Lara's sight, similar to other guns.")
+GS_DEFINE(GAMEPLAY_SETTINGS_ENABLE_GAME_MODES_DESCRIPTION,              "Allows new game plus options to be selected from the new game passport menu.\n\n- New Game+: unlocks all weapons with infinite ammo; enemies have double the HP.\n- Japanese NG: weapons do double damage.\n- Japanese NG+: combination of New Game+ and Japanese NG.")
+GS_DEFINE(GRAPHIC_SETTINGS_ENABLE_GUN_LIGHTING_DESCRIPTION,             "Enables dynamic lighting to be generated for gunshots and explosions, similar to TR2+.")
+GS_DEFINE(GRAPHIC_SETTINGS_ENABLE_FADE_EFFECTS_DESCRIPTION,             "Makes the inventory and pause screen use fade transitions for the semitransparent backdrop, similar to PS1.")
+GS_DEFINE(GAMEPLAY_SETTINGS_FIX_FLOOR_DATA_ISSUES_DESCRIPTION,          "Fixes original issues with floor data/triggers, including the following.\n\n- The untriggered bat in Colosseum.\n- The secret in Great Pyramid that does not register.")
```

Comment:
We should perhaps drop the bullet list as it's now so extensive and covered in detail in the README.

Reply:
Thanks, updated